### ISSUE TITLE
Cannot read property 'message' of null

### ISSUE THREAD

**Issue by ostrolucky** [getsentry/sentry-fullstory#40]
Please see https://sentry.io/share/issue/d718ea2383274a68a93c55dcd7797fea/

**Comment by scefali** [comment]
> https://sentry.io/share/issue/d718ea2383274a68a93c55dcd7797fea/

@ostrolucky What version of the package are you using? This should be fixed in v1.1.3. Here's the PR: https://github.com/getsentry/sentry-fullstory/pull/35

**Comment by ostrolucky** [comment]
It is 1.1.3 actually:

```
npm list | grep fullstory
+-- @fullstory/browser@1.4.4
+-- @sentry/fullstory@1.1.3
```

**Comment by ostrolucky** [comment]
Another Sentry error coming from this library. Related to this one:

https://sentry.io/share/issue/d51f4ef791b74571a944cf809bf8a3d1/

```
TypeError: t.originalException is null
  at getOriginalExceptionProperties(./node_modules/@sentry/fullstory/dist/index.es.js:61:9)
  at processor(./node_modules/@sentry/fullstory/dist/index.es.js:102:74)
  at executor(./node_modules/@sentry/hub/esm/scope.js:68:30)
  at handler(./node_modules/@sentry/utils/esm/syncpromise.js:73:13)
  at _notifyEventProcessors(./node_modules/@sentry/hub/esm/scope.js:61:16)
  at executor(./node_modules/@sentry/hub/esm/scope.js:75:27)
  at handler(./node_modules/@sentry/utils/esm/syncpromise.js:73:13)
  at _notifyEventProcessors(./node_modules/@sentry/hub/esm/scope.js:61:16)
  at executor(./node_modules/@sentry/hub/esm/scope.js:75:27)
  at handler(./node_modules/@sentry/utils/esm/syncpromise.js:73:13)
  at _notifyEventProcessors(./node_modules/@sentry/hub/esm/scope.js:61:16)
  at executor(./node_modules/@sentry/hub/esm/scope.js:75:27)
  at handler(./node_modules/@sentry/utils/esm/syncpromise.js:73:13)
  at _notifyEventProcessors(./node_modules/@sentry/hub/esm/scope.js:61:16)
  at applyToEvent(./node_modules/@sentry/hub/esm/scope.js:343:21)
  at call(./node_modules/@sentry/core/esm/baseclient.js:225:33)
  at _prepareEvent(./node_modules/@sentry/browser/esm/client.js:35:47)
  at executor(./node_modules/@sentry/core/esm/baseclient.js:347:19)
  at handler(./node_modules/@sentry/utils/esm/syncpromise.js:73:13)
  at _processEvent(./node_modules/@sentry/core/esm/baseclient.js:346:16)
  at apply(./node_modules/@sentry/core/esm/baseclient.js:91:14)
  at _invokeClient(./node_modules/@sentry/hub/esm/hub.js:58:39)
  at captureEvent(./node_modules/@sentry/hub/esm/hub.js:184:14)
  at handler(./node_modules/@sentry/browser/esm/integrations/globalhandlers.js:61:28)
  at triggerHandlers(./node_modules/@sentry/utils/esm/instrument.js:77:17)
  at it/tt.onerror(./node_modules/@sentry/utils/esm/instrument.js:433:9)
```

**Comment by scefali** [comment]
@ostrolucky This should be fixed in the newly released version 1.1.4: https://www.npmjs.com/package/@sentry/fullstory/v/1.1.4

If you can confirm, I will close this issue

**Comment by ostrolucky** [comment]
confirmed
